ultimatejobweb/models.py

```diff
@@ -1,17 +1,17 @@
 from django.db import models
-from django.contrib.auth.models import User
-
-
-class user(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    profession = models.CharField(max_length=128)
 
 
 class company(models.Model):
-    UrlCompany = models.URLField(max_length=200)
     CompanyName = models.CharField(max_length=128)
+    CompanySearchUrl = models.URLField()
+    CompanyLogo = models.TextField()
+    FunctionName = models.TextField()
+
 
-    def __str__(self):
-        return self.CompanyName
+class job(models.Model):
+    CompanyName = models.ForeignKey(company, on_delete=models.CASCADE)
+    SearchKey = models.CharField(max_length=128)
```

Comment:
I don't think you want search key here.
This is the list of all the jobs on the company's website.

Reply:
Because if they searched the site for a job by name devops it would give us other names, the search from the database would be based on the search key and not on the names of the jobs we got from the sites

Reply:
How would you know what the search key is?  Your script is downloading all the jobs. There were no searches. One of the jobs it found was Linux Python Developer. 
User A comes into the site and searches for "python", you would expect it to find this job.
User B searches for Linux, you would expect it to find this job.
User C searches for developer, you would also expect it to find this job.
What value are you putting into SearchField?

Reply:
After discussed with Rom & Nuri we decided that at point we'll display all the table to the user.
After the next we'll create a function that will search the user input in our database.